src/widgets/sidebar-estimation-section/SidebarEstimationSection.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 
 import Icon from '../../components/icon/Icon';
+import DurationTime from '../../components/format/DurationTime';
```

Comment:
All formatting components should be imported with named import like this:

```jsx
import { DurationTime } from '../../components/format';
```

For this add **components/format/index.js** that will re-export all necessary modules like this

```js
export { default as DurationTime } from './DurationTime';
```